### DISCUSSION TITLE
Im trying to make a shooter game but with blocks im trying to get to where the player is a block

### DISCUSSION THREAD

**Discussion by sosa360** [HaxeFlixel/flixel#2498] [Q&A]
i'm working on a project here you shoot blocks and i don't know here to start can i get some feedback

the game looks something like this 
https://www.newgrounds.com/portal/view/710918

NO OFFENSE TO ANY RACE

**Comment by RichardBray** [comment]
Nice small game. I like the sounds and the words. I have a feeling this one was inspired by recent events 😁

**Reply by Geokureli** [reply]
FYI Richard, this game is an example of what he's trying to make. The game was made by NinjaMuffin (FNF) and it's a sequel to this
https://www.newgrounds.com/portal/view/708841

Maybe @ninjamuffin99 can share the source? Github removed the old repo, likely because it's called "whites only"

**Reply by ninjamuffin99** [reply]
yep was removed by github, i still have the source on my computer, i forgot to rehost it tho

**Reply by sosa360** [reply]
lol I been wanted it for a long time

**Reply by sosa360** [reply]
i am a pain in my butt

**Reply by RichardBray** [reply]
Ah I see, I should have read the message properly first 🤦 
I don't know how familiar with HF but I think the tutorial on the site is a good place to start if you haven't checked it out already:
https://haxeflixel.com/documentation/tutorial/
